pkg/registry/apis/provisioning/usage.go

```diff
@@ -0,0 +1,54 @@
+package provisioning
+
+import (
+	"context"
+
+	"k8s.io/apimachinery/pkg/labels"
+
+	"github.com/grafana/grafana/pkg/storage/unified/resource"
+)
+
+func (b *APIBuilder) collectProvisioningStats(ctx context.Context) (map[string]any, error) {
+	m := map[string]any{}
+	if b.unified == nil {
+		return m, nil
+	}
+
+	// FIXME: hardcoded to "default" for now -- it works for single tenant deployments
+	// we could discover the set of valid namespaces, but that would count everything for
+	// each instance in cloud.
+	//
+	// We could get namespaces from the list of repos below, but that could be zero
+	// while we still have resources managed by terraform, etc
+	ns := "default"
+	count, err := b.unified.CountManagedObjects(ctx, &resource.CountManagedObjectsRequest{
+		Namespace: ns,
+	})
+	if err != nil {
+		return m, err
+	}
+	counts := make(map[string]int, 10)
+	for _, v := range count.Items {
+		counts[v.Kind] = counts[v.Kind] + int(v.Count)
+	}
+	for k, v := range counts {
```

Comment:
why is it  "repo" only and not "repository"? 

show we use `provisioning` instead of repository for the manager kind? as it's the overall feature.